tensorflow/contrib/grid_rnn/python/ops/grid_rnn_cell.py

```diff
@@ -440,8 +534,8 @@ def _propagate(dim_indices, conf, cells, c_prev, m_prev, new_output, new_state,
   if conf.num_dims > 1:
     ls_cell_inputs = [None] * (conf.num_dims - 1)
     for d in conf.dims[:-1]:
-      ls_cell_inputs[d.idx] = new_output[d.idx] if new_output[
-          d.idx] is not None else m_prev[d.idx]
+      ls_cell_inputs[d.idx] = new_output[d.idx] \
```

Comment:
we don't allow python line continuations with a backslash.  please fix these everywhere to something like:

ls_cell_inputs[d.idx] = (
  new_output[d.idx] if new_output... is not None
  else m_prev[d.idx])